app/controllers/authenticate_controller.rb

```diff
@@ -120,10 +120,11 @@ def authenticator_input
   def k8s_inject_client_cert
     # TODO: add this to initializer
     Authentication::AuthnK8s::InjectClientCert.new.(
-      conjur_account:   ENV['CONJUR_ACCOUNT'],
-      service_id:       params[:service_id],
-      client_ip:        request.ip,
-      csr:              request.body.read,
+      conjur_account:                ENV['CONJUR_ACCOUNT'],
+      service_id:                    params[:service_id],
+      client_ip:                     request.ip,
+      csr:                           request.body.read,
+      cert_installation_path:        request.headers["Certificate-Install-Path"],
```

Comment:
should this be `Certificate-Installation-Path`?

@shulifink can you review this? it is the path in which the Conjur server will install the certificate inside the client container.

Reply:
It's funny how each one has half abbreviated and half a full word.
Why not just use "Cert_Installation_Path" line the option?

Reply:
but that's funny in the same way because now `Cert` is half abbreviated and `Installation` is a full word. I would use either `Cert-Install-Path` or `Certificate-Installation-Path`.

Regarding `cert_installation_path` - that's not customer facing but just a code variable so i wouldn't bother with its naming.